src/Whoops/Run.php

```diff
@@ -293,6 +293,24 @@ public function handleException(Exception $exception)
     }
 
     /**
+     * This is a private implementation detail, you are not supposed to call this.
+     */
+    final public function handleExceptionFromPHP($exception)
```

Comment:
I guess we should rename the parameter to `$throwable` because we dont know yet it actuall is a `exception`